cluster-stamp.bicep

```diff
@@ -63,9 +63,6 @@ var acrPullRole = '${subscription().id}/providers/Microsoft.Authorization/roleDe
 var managedIdentityOperatorRole = '${subscription().id}/providers/Microsoft.Authorization/roleDefinitions/f1a07417-d97a-45cb-824c-7a7467783830'
```

Comment:
any desire to replicate the same design for this one and others?

Reply:
Not in this pass, as these didn't seem to have problems (still don't really know why) - and I'm keeping the changes scope to just those that I know had the issue.  But yes, I think it would make sense to see if we can do this with the rest of them.